src/iOS/Xamarin.iOS/Samples/Map/ChangeAtmosphereEffect/ChangeAtmosphereEffect.cs

```diff
@@ -79,15 +79,13 @@ public override void LoadView()
             _atmosphereEffectPicker = new UISegmentedControl("Realistic", "Horizon only", "None");
             _atmosphereEffectPicker.TranslatesAutoresizingMaskIntoConstraints = false;
             _atmosphereEffectPicker.SelectedSegment = 1;
-            _atmosphereEffectPicker.TintColor = UIColor.White;
-            _atmosphereEffectPicker.BackgroundColor = UIColor.Black;
+            _atmosphereEffectPicker.BackgroundColor = UIColor.White;
```

Comment:
This is a good change. I recommend updating the feature layer extrusion sample to have a white background as well.